src/main/java/com/exactpro/th2/common/event/Event.java

```diff
@@ -207,6 +208,15 @@ public Event bodyData(IBodyData bodyData) {
         return this;
     }
 
+    /**
+     * Adds passed collection of body data
+     * @return current event
+     */
+    public Event bodyData(Collection<IBodyData> bodyDataCollection) {
```

Comment:
Otherwise, we won't be able to add a collection with a certain type. E.g.
```java
class Body implements IBodyData {}

List<IBodyData> list = List.of(new Body());
new Event().bodyData(list); // works fine

List<Body> list = List.of(new Body());
new Event().bodyData(list); // compilation error
```
To fix the error we should change the method signature
```suggestion
    public Event bodyData(Collection<? extends IBodyData> bodyDataCollection) {
```